### ISSUE TITLE
Create a prioritized list of Autons

### ISSUE THREAD

**Issue by witcpalek** [Team302/2023ChargedUp#34]
Need a list of Autons with their priority.

I think we may have already done this, but we need to document those here as well as create separate stories for each one.

**Comment by Tan560** [comment]
We need a 2 Cone Auton with Balancing at the end from each scoring zone (Wall, Community, and HP) for each alliance color. Then once this is complete we will want to make a copy for each without balance. This is a lot of so I don't know if we want to talk tonight and break it out into smaller ones?

**Comment by Tan560** [comment]
I also think we want to think about how we are going to do autons. I think we should make it so that we don't need to select a cube or cone. I think it should be a Auton mode that when you are at a point ready to score. It looks at the color in the gripper and then goes in and automatically goes to position based on that info and field position/april tags

**Comment by witcpalek** [comment]
@Tan560 @SierraManriquez How close are we to having this one done?

**Comment by Tan560** [comment]
@witcpalek I talked with Sierra 2 days ago and we came up with this list.
1. Start Coop Grid ,1 Game Piece(We want it to be cone, but might want to switch it to cube since we are faster at picking up cones off the ground) and balance
2. Start HP Grid, score 1 GP, and then drive around and balance.
3. Start Wall Grid, score 1 GP, and then drive around and balance.
4. Start HP Grid, score 2 GP, and then drive around and balance.
5. Start Wall Grid, score 2 GP, and then drive around and balance.
I think if we can get the first 3 done we will be really competitive. Then move onto scoring 2 GP, but we have to get 1-3 done by Week 1
